app/_includes/events.html

```diff
@@ -7,6 +7,14 @@
 			<h1 class="title feature-header">{{site.data[locale].events.title}}</h1>
 		</div>
 	</div>
+	<div class="row" style="text-align: center">
+		<a href="https://docs.google.com/forms/d/e/1FAIpQLScdvsMHLor8mrDrHa-BQs2XzFb96vIrx3u--LHXWd1g7RncPg/viewform">
+			<div class="btn btn-blue arw">{{site.data[locale].events.register}}
+				<img src="{{ site.baseurl }}/assets/graphics/meta/Arrow.svg" alt="learnnow" />
```

Comment:
The alt attribute should tell a screenreader about the image that cannot be seen. Since this image does not add to the message of the link, I suggest adding an empty alt-attribute so screenreader are not confused `alt=""`